packages/multicast_dns/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright 2014 The Chromium Authors. All rights reserved.
```

Comment:
2019?

Reply:
No, if it was originally written in 2014, even if we're modifying it and copying it, we should leave it as 2014.